plans/install.pp

```diff
@@ -142,6 +142,16 @@
     stagingdir                       => $stagingdir,
     final_agent_state                => $final_agent_state,
   )
+  # Log parameters for debugging 
```

Comment:
I'd move this up to be the first thing we do in the plan, but looks good other than that